integrations/standalone/tests/pageobjects/DataClassEditor.ts

```diff
@@ -1,10 +1,20 @@
-import { expect, type Page } from '@playwright/test';
+import { expect, type Locator, type Page } from '@playwright/test';
+import { Button } from './Button';
+import { Settings } from './Settings';
 
 export class DataClassEditor {
   readonly page: Page;
+  readonly locator: Locator;
+  readonly masterPanel: Locator;
+  readonly detailsToggle: Button;
+  readonly settings: Settings;
 
   constructor(page: Page) {
     this.page = page;
+    this.locator = page.locator(':root');

```

Comment:
?

Reply:
I don't know, I took this over from the variables editor. I guess I can replace all the references to this locator with the page itself?

Reply:
As this is the "root" page object, I don't think you need to have this locator field at all. What I mean is that all the other things can directly be created from the page object.